src/screens/main/portfolio/index.js

```diff
@@ -258,19 +259,33 @@ const styles = StyleSheet.create({
     width: '82%',
     height: Dimensions.get('window').height * 0.082,
   },
-  footerContainer: {
+  footerGrandparentContainer: {
+    alignItems: 'center',
+    marginBottom: '5%',
+    marginTop: '5%',
+  },
+  footerParentContainer: {
     alignItems: 'center',
-    justifyContent: 'flex-end',
-    flex: 1,
   },
-  textFooter : {
-    fontFamily: "WorkSans-Regular",
+  textFooter: {
+    fontFamily: 'WorkSans-Regular',
     fontSize: RF(1.7),
-    marginBottom: '5%',
-    alignItems: 'center' ,
     color: '#c0c0c0',
     letterSpacing: 0.5
-  }
+  },
```

Comment:
remove unused styles before pushing to master

Reply:
fixed